arelle/plugin/validate/EFM/Filing.py

```diff
@@ -1149,7 +1150,10 @@ def isADR(f):
                             if name.endswith(":*") and validation == "(supported-taxonomy)": # taxonomy-prefix filter
                                 txPrefix = name[:-2]
                                 ns = deiDefaultPrefixedNamespaces.get(txPrefix)
-                                if ns:
+                                # Its possible that extension concepts could have prefixes that match `cef` of `vip`
+                                # and trip this validation so we exclude all extension namespaces by making sure the
+                                # qname namespace matches known SEC domains.
+                                if ns and secDomainPattern.match(ns):
```

Comment:
It would be better to filter the namespaces and prefixes in `deiDefaultPrefixedNamespaces`. The name implies that it's only DEI namespaces, but that's clearly not the case.